strax/context.py

```diff
@@ -1014,7 +1017,17 @@ def get_array(self, run_id: ty.Union[str, tuple, list],
                 max_workers=max_workers,
```

Comment:
Since this function will be applied outside of get_iter it wont be included in the computation time of the progressbar. Now, I also understand why progressbars always tend to take ages for the last 0.2 % ;-) Maybe save thing would be something like if self.config['apply_data_function'] switch off progressbar.

Reply:
We could do that, on the other hand, it's fine if the progress bar reflects the loading. What happens after that can fal outside of the scope of the progress bar. For example, the select_runs also contains several steps that take different amounts of time.